docs/environment.yaml

```diff
@@ -1,23 +1,26 @@
-name: docs
+name: pontibus-docs
 channels:
 - conda-forge
 dependencies:
-- autodoc-pydantic<2.0
-- openfe>=1.1.0
+- autodoc-pydantic>=2.0
+# - openfe>=1.1.0
 - packaging
 - plugcli
-- python=3.10
-- sphinx<7
+- openff-toolkit
+- sphinx
 - sphinx-click
 - gitpython
 - libsass
 - nbsphinx
 - nbsphinx-link
 - myst-parser
+- konnektor
+- lomap2>=3.0.0
```

Comment:
we need these at least until we're building from a conda release of openfe - maybe theyll be implicit then?